changelog/@unreleased/pr-16.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: Add an option to configure the default health state value
```

Comment:
Can you expand on this a little bit so that usage can be determined from release notes? Something like:

```
Adds the WithFailingHealthStateValue ErrorOption that allows the default health state value [reported by a check?] to be configured.

Prior to this, [a health check? the windowed health check?] would always report an ERROR state by default until its status was computed, but this option allows a different state to be reported in this case instead.
```

Wording above isn't perfect (I still don't understand all the nuances), but I think that's the level of detail that would be helpful.

Reply:
I updated the changelog with some parts of your description. Let me know what you think